R/dash.R

```diff
@@ -249,23 +249,23 @@ Dash <- R6::R6Class(
         #
         # https://cran.r-project.org/doc/FAQ/R-FAQ.html#Others:
         callback_args <- list()
-        
+
         for (input_element in request$body$inputs) {
           if(is.null(input_element$value))
             callback_args <- c(callback_args, list(list(NULL)))
           else
-            callback_args <- c(callback_args, input_element$value)
+            callback_args <- c(callback_args, list(input_element$value))
```

Comment:
I should've caught this when I wrapped `list(NULL)` in another `list`, since this is problematic for non-NULL `value` fields also.